cryptol-remote-api/python/cryptol/connection.py

```diff
@@ -396,6 +396,14 @@ def safe(self, expr : Any, solver : solver.Solver = solver.Z3, *, timeout:Option
         self.most_recent_result = CryptolSafe(self, expr, solver, timeout)
         return self.most_recent_result
 
+    def modules(self, *, timeout:Optional[float] = None) -> argo.Command:
+        """Discover the list of modules and submodules currently in scope in the current context.
+
+        :param timeout: Optional timeout for this request (in seconds)."""
+        timeout = timeout if timeout is not None else self.timeout
+        self.most_recent_result = CryptolModules(self, timeout)
+        return self.most_recent_result
```

Comment:
Could you add a test case to the [Python test suite](https://github.com/GaloisInc/cryptol/tree/670262219d9e1b001c2130cc81c3ce3680cd8029/cryptol-remote-api/python/tests/cryptol) that demonstrates that this command works as expected? (Something as simple as retrieving the docstring for a simple module and checking that it matches an expected value should suffice.)